### DISCUSSION TITLE
My Bluetooth headset's microphone is not working in TeamTalk.

### DISCUSSION THREAD

**Discussion by emrerencber** [BearWare/TeamTalk5#1113] [Q&A]
Hello, dear TeamTalk developers and users who have knowledge on the topic. I use a headset called Anker Q10. In the accessibility client I can use the headset as a speaker, but when I select it as a microphone, the Teamtalk sound is completely gone. TeamTalk stutters and freezes. Speaker sound goes away. How can I solve this problem?

**Comment by bear101** [comment]
What platform is this? Windows? Also is it still an issue?

**Reply by alessio0292** [reply]
please @bear101, please i repeat, read my 18 discussion go to autor and put my name and you read all my disscussion very important that i did here. Ok Bjoern @bear101?

**Comment by emrerencber** [comment]
Yes. İt is a Windows's problem. But ı thing this problem coming headset.

2022-02-24 23:14 GMT+03:00, Bjørn Damstedt Rasmussen ***@***.***>:
